birdhouse/scripts/bootstrap-testdata

```diff
@@ -7,7 +7,7 @@
 
 
 if [ -z "$DATASET_ROOT" ]; then
-    DATASET_ROOT="/data/datasets"
+    DATASET_ROOT="${DATA_PERSIST_ROOT:-/data}/datasets"
```

Comment:
Can't remember, but if this script do not source defualt.env and/or env.local then yes you should duplicate the default value here. Otherwise please avoid duplication when possible.

Reply:
idem as other